dot/sync/chain_processor_integration_test.go

```diff
@@ -206,7 +209,7 @@ func TestChainProcessor_HandleJustification(t *testing.T) {
 	d, err := types.NewBabeSecondaryPlainPreDigest(0, 1).ToPreRuntimeDigest()
 	require.NoError(t, err)
 	digest := types.NewDigest()
-	err = digest.Add(*d)
```

Comment:
Why has this changed?

`digest.Add` takes an interface, but I think if you look at its code, you will find that it accepts a value.